packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Models/Types/ModelTypeProvider.cs

```diff
@@ -13,13 +13,20 @@ namespace Microsoft.Generator.CSharp
     public sealed class ModelTypeProvider : TypeProvider
     {
         private readonly InputModelType _inputModel;
-
         public override string Name { get; }
 
+        public bool IsPropertyBag { get; }
```

Comment:
I added this property since [in autorest ](https://github.com/Azure/autorest.csharp/blob/f9b784745c8531520eb5bf2686dec2b1f74448ae/src/AutoRest.CSharp/Common/Output/Models/Types/ModelTypeProvider.cs#L574 ) we skip building the json serialization if a model is a "property bag". Is this still a valid thing we want to check to determine if we want json serialization? If so, I think we will need it to be public to skip constructing a serialization provider for a model? Otherwise, I can remove it.

Reply:
I've removed this for now.